tests/plugins/tableresize/manual/maximizeddivarea.html

```diff
@@ -0,0 +1,27 @@
+<div id="editor">
+	<table border="1" cellpadding="1" cellspacing="1" style="width:600px">
+		<thead>
+			<tr>
+				<th scope="row" style="width:150px">Header</th>
+				<th scope="col" style="width:300px" colspan="2">Header</th>
+				<th scope="col" style="width:150px">Header</th>
+			</tr>
+		</thead>
+		<tbody>
+			<tr>
+				<th scope="row" style="width:150px">Header</th>
+				<td style="width:150px">not header</td>
+				<td style="width:150px">not header</td>
+				<td style="width:150px">not header</td>
+			</tr>
+			<tr>
+				<th scope="row" style="width:300px" colspan="2">Header</th>
+				<td style="width:300px" colspan="2">not header</td>
+			</tr>
+		</tbody>
+	</table>
+</div>
+
+<script>
+	CKEDITOR.replace( 'editor' );
```

Comment:
It would be good to have all 3 editor types available in this test, so:

*   Classic one (you may use `removePlugins: divarea` to make sure it's iframe based),
*   `divarea` - already present,
*   Inline one - using `CKEDITOR.inline`.

This way we may catch any strange behaviors in any editor types (unlikely to happen here, but it's a good practice anyway to have coverage for all editor types).

Reply:
Well, there is a slight issue with `inline` editor since [`maximze` plugin is disabled there](https://github.com/ckeditor/ckeditor4/blob/ffd8a623e0e74cce75eed8876af920487f3b721c/plugins/maximize/plugin.js#L101-L103) (as it doesn't make much sense there).

So maybe let's leave classic and divarea editor as is and for inline editor lets go with some absolute positioned wrapper with high `z-index` (so we will have also case mentioned in #909) covered. It should be probably a separate manual test then.